LICENSE

```diff
@@ -1,4 +1,4 @@
-TypeScript ESLint
+typescript-eslint
 
 Originally extracted from:
```

Comment:
Side note - I wonder if we should just standardise everything on true MIT? We're a hodge-lodge right now.